sentry-kotlin-multiplatform/sentry_kotlin_multiplatform.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'sentry_kotlin_multiplatform'
-    spec.version                  = '0.0.1'
+    spec.version                  = '0.0.1-alpha.2'
```

Comment:
```suggestion
    spec.version                  = '0.0.1'
```

Reply:
Yes, it auto-increments on every build and forgot to unstage it. Maybe I should add it to gitignore from now on?

Reply:
maybe remove the part that makes it auto-increment?

Reply:
seems like pinning the version to `0.0.1` in the cocoapods gradle plugin works